rest_api/stdeb.cfg

```diff
@@ -0,0 +1,2 @@
+[DEFAULT]
+Depends3: python3-sawtooth-sdk
```

Comment:
Why do we suddenly need to add this file, if the SDK is already listed in setup.py?

Reply:
`python3-sawtooth-rest-api` is no longer getting added as a dependency in the deb package. This file seems to be the standard way to add dependencies to the deb package when they are not added automatically.